src/main/java/Utils.java

```diff
@@ -0,0 +1,11 @@
+public class Utils {
+    public static void echo(String input) {
+        System.out.println(input + "\n");
+        printDivider();
+
+    }
+    public static void  printDivider(){
```

Comment:
Perhaps can take note of the spacing? I noticed the spacing is inconsistent in other parts too.
```suggestion
    public static void printDivider() {
```